clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -120,6 +120,10 @@ Changes in existing checks
   <clang-tidy/checks/modernize/use-std-print>` check to support replacing
   member function calls too.
 
+- Improved :doc:`performance-avoid-endl
+  <clang-tidy/checks/performance/avoid-endl>` check by fixing incorrect
+  message.
+
```

Comment:
This feels a bit terse to me. (possible change would depend on the resolution to the other comment)